### PR TITLE
docs: Improve tutorial

### PR DESCRIPTION
**Pull request by jamyct**
### Description of your changes

Improve tutorial for Migration with Override DR

Fixes #

I have:

- [x] Run `make reviewable` to ensure this PR is ready for review.

### CODE REVIEW

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 1]
```diff
@@ -318,13 +318,13 @@ spec:
   strategy:
     type: RollingUpdate
 ```
-Update the `crp.yaml` to reflect selecting clusters with higher node-count and apply it:
+Update the `crp-availability.yaml` to reflect selecting clusters with higher node-count and apply it:
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:321]: hyperlink `crp-availability.yaml` with link to test file

**Review by britaniar — CHANGES_REQUESTED** [review]
Add links within the tutorial to the correlated files, otherwise the user will not know they exist for this purpose.

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ The following resources are currently deployed in the WestUS clusters:
 
 #### Service
 
-Service test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-service.yaml] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-service.yaml)
+Service test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-service.yaml] (./tesfiles/nginx-service.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md]:
```Suggestion
Service test file located here: [nginx-service.yaml](./tesfiles/nginx-service.yaml)
```

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ Summary:
 
 #### Deployment
 
-Deployment test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-deployment.yaml] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-deployment.yaml)
+Deployment test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/nginx-deployment.yaml] (./tesfiles/nginx-deployment.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md]:
```Suggestion
Deployment test file located here: [nginx-deployment.yaml](./tesfiles/nginx-deployment.yaml)
```

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ Summary:
 
 #### ClusterResourcePlacement
 
-CRP Availability test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml)
+CRP Availability test file located here: [https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml] (./tesfiles/crp-availability.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md]:
```Suggestion
CRP Availability test file located here: [crp-availability.yaml](./tesfiles/crp-availability.yaml)
```

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 5]
```diff
@@ -327,13 +327,13 @@ spec:
   strategy:
     type: RollingUpdate
 ```
-Update the [`crp-availability.yaml`] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml) to reflect selecting clusters with higher node-count and apply it:
+Update the [`crp-availability.yaml`] (./tesfiles/crp-availability.yaml) to reflect selecting clusters with higher node-count and apply it:
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:330]:
```Suggestion
Update the [`crp-availability.yaml`](./tesfiles/crp-availability.yaml) to reflect selecting clusters with higher node-count and apply it:
```

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 6]
```diff
@@ -327,13 +327,13 @@ spec:
...
 ```
-Update the [`crp-availability.yaml`] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml) to reflect selecting clusters with higher node-count and apply it:
+Update the [`crp-availability.yaml`] (./tesfiles/crp-availability.yaml) to reflect selecting clusters with higher node-count and apply it:
 ```bash
 kubectl apply -f crp-availability.yaml
 ```
 
 ### Results
-After applying the updated [`crp-availability.yaml`] (https://github.com/Azure/fleet/blob/main/docs/tutorials/tesfiles/crp-availability.yaml), the Fleet will schedule the application on the available clusters in WestEurope as they each have 3 nodes.
+After applying the updated [`crp-availability.yaml`] (./tesfiles/crp-availability.yaml), the Fleet will schedule the application on the available clusters in WestEurope as they each have 3 nodes.
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:336]:
```Suggestion
After applying the updated [`crp-availability.yaml`](./tesfiles/crp-availability.yaml), the Fleet will schedule the application on the available clusters in WestEurope as they each have 3 nodes.
```

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 7]
```diff
@@ -15,6 +15,9 @@ Due to a sudden increase in traffic and resource demands in your WestUS clusters
 The following resources are currently deployed in the WestUS clusters:
 
 #### Service
+
+Service test file located [here](./testfiles/nginx-service.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:19]:
```Suggestion
> Note: Service test file located [here](./testfiles/nginx-service.yaml).
```
Move to the line 18.

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 8]
```diff
@@ -36,6 +39,9 @@ Summary:
 - It targets pods with the label app: nginx and forwards traffic to port 80 on the pods.
 
 #### Deployment
+
+Deployment test file located [here](./testfiles/nginx-deployment.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:43]:
```Suggestion
> Note: Deployment test file located [here](./testfiles/nginx-deployment.yaml).
```
Move to the line 42.

File: docs/tutorials/MigrationWithOverrideDR.md [hunk 9]
```diff
@@ -67,6 +73,9 @@ Summary:
 - The pods are labeled with `app: nginx` and expose port 80.
 
 #### ClusterResourcePlacement
+
+CRP Availability test file located [here](./testfiles/crp-availability.yaml)
```
Comment (britaniar) [inline docs/tutorials/MigrationWithOverrideDR.md:77]:
```Suggestion
> Note: Deployment test file located [here](./testfiles/nginx-deployment.yaml).
```
Move to the line above 76.

File: docs/tutorials/testfiles/crp-availability.yaml [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+  name: crp-availability
+spec:
+  policy:
+    placementType: PickN
+    numberOfClusters: 2
+    affinity:
+      clusterAffinity:
+        requiredDuringSchedulingIgnoredDuringExecution:
+          clusterSelectorTerms:
+            - propertySelector:
```
Comment (britaniar) [inline docs/tutorials/testfiles/crp-availability.yaml:13]: I think this should be the original crp where this crp selects the labels instead.
```
 - labelSelector:
      matchLabels:
        fleet.azure.com/location: westus
```

**Review by britaniar — COMMENTED** [review]
Please similarly link the cluster resource override yaml as well.

File: docs/tutorials/testfiles/crp-availability.yaml (hunk 10 above)
Comment (jamyct) [reply docs/tutorials/testfiles/crp-availability.yaml]: I've updated this section to match the original code snippet

**Review by britaniar — APPROVED** [review]

**Pull request merged**